plugins/modules/vm_list_group_by_clusters.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Ansible Cloud Team (@ansible-collections)
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: vm_list_group_by_clusters
+short_description: Returns information about the virtual machines grouping by clusters and folders
+description:
+    - Returns information about the virtual machines grouping by clusters and folders.
+author:
+    - Ansible Cloud Team (@ansible-collections)
+options:
+  detailed_vms:
+    default: true
+    description:
+      - Wither or not get the vms detailed.
+    type: bool
+extends_documentation_fragment:
+- vmware.vmware.vmware_rest_client.documentation
+
+'''
+
+EXAMPLES = r'''
+- name: Gather list of VMs
+  vmware.vmware.vm_list_group_by_clusters:
+    hostname: "https://vcenter"
+    username: "username"
+    password: "password"
+
+- name: Gather list of VMs in clusterA
```

Comment:
I think this description is wrong. The module don't support grouping by specific cluster.

Reply:
removed that example